src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxService.cs

```diff
@@ -305,6 +307,14 @@ await requestContext.SendResult(new DacFxOptionsResult()
             }
         }
 
+        public async Task HandleParseTSqlScriptRequest(ParseTSqlScriptRequestParams requestParams, RequestContext<ParseTSqlScriptResult> requestContext)
+        {
+            await requestContext.SendResult(new ParseTSqlScriptResult()
```

Comment:
should this be in a try just in case `DacTableDesigner.ScriptContainsCreateTableStatements()` throws an error?

Reply:
done. good catch